app/views/support_interface/providers/show.html.erb

```diff
@@ -38,6 +38,9 @@
   <%= render(SupportInterface::ProviderCoursesTableComponent.new(provider: @provider, courses: @provider.accredited_courses)) %>
 <% end %>
 
+<h2 class='govuk-heading-l'>Course choices & vacancies</h2>
+<%= render(SupportInterface::CourseChoicesTableComponent.new(course_options: @provider.course_options.includes(:course, :site))) %>
```

Comment:
Suggest we do the AR query methods in the controller, assigning a separate `@course_options`.

Reply:
👍 Updated!